sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client_aad_async.py

```diff
@@ -392,7 +392,7 @@ async def test_list_revisions_fields(self, appconfiguration_endpoint_string):
     async def test_list_revisions_correct_etag(self, appconfiguration_endpoint_string):
         await self.set_up(appconfiguration_endpoint_string, is_aad=True)
         to_list_kv = self.create_config_setting()
-        custom_headers = {"If-Match": to_list_kv.etag}
+        custom_headers = {"If-Match": to_list_kv.etag or ""}
```

Comment:
Why we need this?

Reply:
`aiohttp` iterates over the headers of each request and attempts to do a string concatenation of their keys and values. If we send a non-string header value -- which we were doing here, because `to_list_kv.etag` returns `None` -- it works with `asyncio` but breaks `aiohttp`.

I'm not sure what value this header adds since the value is consistently `None`, but the test isn't transport-independent without the change.